src/commandHandlers/help.ts

```diff
@@ -10,24 +10,26 @@ export const command = async (arg: [string, string], embed: MessageEmbed) => {
     triggers.find((trigger) => trigger === arg[1])
   );
 
-  embed.setTitle('Help commands').setDescription('Lists the command available');
-
   if (matching) {
-    embed.addField(
-      `${COMMAND_PREFIX}${matching.triggers[0]}`,
-      `${matching.description}\nUsage: ${COMMAND_PREFIX}${matching.usage}`,
-      false
-    );
+    const blocky = (arg: string) =>
+      '`' + `${COMMAND_PREFIX}` + `${arg}`.trim() + '`';
+    const text = `${matching.usage}`.split('or');
+    const formatted = text.map(blocky).join(' or ');
```

Comment:
what do you think about changing the names of these variables, to be more explicit on what they hold and their purpose?

For the `blocky` function I think we can refactor the name and make it more generic, since the `format` function is similar. What do you think of this: 
```ts
export const command = async (arg: [string, string], embed: MessageEmbed) => {
  //...
  return embed;
};

const mapToCodeBlock = (arg: string) => '`' + `${COMMAND_PREFIX}` + `${arg}`.trim() + '`';
```
Here is my suggestion for the other variables:
```suggestion
    const usageTextOptions = matching.usage.split('or');
    const formatted = usageTextOptions.map(mapToCodeBlock).join(' or ');
```

Reply:
btw I removed the template string on `${matching.usage}` since that field is already a `string`, so it's not necessary 😃